composer.json

```diff
@@ -6,11 +6,11 @@
   "minimum-stability": "dev",
   "prefer-stable": true,
   "require": {
-    "twig/twig": "~1"
+    "twig/twig": "~1.0"
   },
   "require-dev": {
-    "phpunit/phpunit": "~5||~6.0",
-    "openeuropa/code-review": "~0.3"
+    "phpunit/phpunit": "~5.0||~6.0",
```

Comment:
Let's also update this to remove the double pipe, this is not according to the Composer specification. Let's make it `~5.0|~6.0`.